archive/terraform/ecr.tf

```diff
@@ -0,0 +1,48 @@
+data "aws_ecr_repository" "ecr_repository_nginx_services" {
+  name = "uk.ac.wellcome/nginx_services"
+}
+
+module "ecr_repository_archivist" {
+  source = "git::https://github.com/wellcometrust/terraform.git//ecr?ref=v1.0.0"
+  name   = "archivist"
+}
```

Comment:
Is it intentional to lose `uk.ac.wellcome` in the names?

Reply:
I think V1 of the modules prepends that for you.